chore: release v0.8.0

DominicKramer opened the pull request:
The draft release notes are in the GitHub UI.

ofrobots commented:
No need for Node 4 examples in the changelog anymore.

ofrobots commented:
Any reason not to include ee8265758c02a9d9bb80b444801ca0da1435f247 in the release?

DominicKramer commented:
@ofrobots I generated the commit logs last Friday, which is why it didn't include the newest commits.  I added them.  I also updated the changelog to exclude Node 4 examples.
